xsd/netex_framework/netex_reusableComponents/netex_vehicleType_support.xsd

```diff
@@ -503,10 +503,15 @@ Rail transport, Roads and Road transport
 			<xsd:enumeration value="cycle"/>
 			<xsd:enumeration value="pedalCycle">
 				<xsd:annotation>
-					<xsd:documentation/>
+					<xsd:documentation>Cycle without motor.</xsd:documentation>
```

Comment:
```suggestion
					<xsd:documentation>Regular bicycle without motor or pedal support.</xsd:documentation>
```

Reply:
but it is pedalCycle... and cycle I see as parent category

Reply:
The first proposal was right, the second not, in my view. See #45, last discussion.